core/views.py

```diff
@@ -1,5 +1,13 @@
 from django.views.generic import TemplateView
+from triage.helpers import TriageAnswersManager
 
 
 class LandingPagelView(TemplateView):
     template_name = 'core/landing-page.html'
+
+    def get_context_data(self, *args, **kwargs):
+        answer_manager = TriageAnswersManager(self.request)
+        has_completed_triage = answer_manager.retrieve_answers() != {}
```

Comment:
maybe it's worth adding it as a context processor? Cause that info will be needed on multiple pages

Reply:
that would be quite implicit and really it's only used on two views